tests/Fable.Tests/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+// Note this only includes basic configuration for development mode.
+// For a more comprehensive configuration check:
+// https://github.com/fable-compiler/webpack-config-template
+
+var path = require("path");
+
+module.exports = {
+    mode: "development",
+    entry: "./Fable.Tests.fsproj",
+    output: {
+        path: path.join(__dirname, "./public"),
+        filename: "bundle.js",
```

Comment:
Just trying to catch up with Fable :smiley: — `bundle.js` contains the Fable-compiled Hedgehog?

Reply:
Yes. It is defined in webpack.config.json in which file fable-compiler will save the compiled/transpiled code.